GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -95,6 +75,18 @@ - (nullable NSString *)openIDRealm {
   return grantedScopes;
 }
 
+- (GIDConfiguration *)configuration {
+  __block GIDConfiguration *configuration;
```

Comment:
The [`__block` storage type](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Blocks/Articles/bxVariables.html) will only persist `configuration` for the life of the block you define below.

Reply:
Fixed.